### PR TITLE
Refactor Address to Station

### PR DESCRIPTION
**Pull request by mynameizzhafeez**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -844,7 +844,7 @@ The `ArgumentMultimap` utilises a `HashMap` to store an `ArrayList<String>` of a
 
 #### **Prefix**
 
-The `Prefix` is an `enum` consisting of `n/` ,`a/`, `p/`, `t/`, `e/`, `g/`, `m/` and a blank `Prefix` which is an empty String. The Prefixes listed previously correspond to [Name](#name), [Address](#address), [Phone](#phone), [Telegram Handle](#telegram-handle), [Email](#email), [Group Tags](#group-tag) and [Module Tags](#module-tag)).
+The `Prefix` is an `enum` consisting of `n/` ,`a/`, `p/`, `t/`, `e/`, `g/`, `m/` and a blank `Prefix` which is an empty String. The Prefixes listed previously correspond to [Name](#name), [Station](#station), [Phone](#phone), [Telegram Handle](#telegram-handle), [Email](#email), [Group Tags](#group-tag) and [Module Tags](#module-tag)).
```
Comment (russelltankaimin) [inline docs/DeveloperGuide.md:847]: You missed `/a` here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -315,11 +315,11 @@ Rules: `EMAIL` should be of the form `local@domain`, where
 * `domain` {can someone fill up}<br>
   Example: `e/wenli@gmail.com`
 
-### Home Address
+### Home Station
 
 Description: The name of the **closest MRT station** to their home.<br>
-Pattern: `a/ADDRESS`<br>
-Rules: `ADDRESS` should be the name of a valid MRT station.<br>
+Pattern: `a/STATION`<br>
```
Comment (russelltankaimin) [inline docs/UserGuide.md:320]: 'a/' here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -315,11 +315,11 @@ Rules: `EMAIL` should be of the form `local@domain`, where
...
+Rules: `STATION` should be the name of a valid MRT station.<br>
 Example: `a/Boon Lay`
```
Comment (russelltankaimin) [inline docs/UserGuide.md:322]: `/a` here

**Review by russelltankaimin — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (mynameizzhafeez) [reply docs/DeveloperGuide.md]: Alright. Done.

File: docs/UserGuide.md (hunk 2 above)
Comment (mynameizzhafeez) [reply docs/UserGuide.md]: Resolved.

File: docs/UserGuide.md (hunk 3 above)
Comment (mynameizzhafeez) [reply docs/UserGuide.md]: Gotcha.

**Pull request merged**